build.gradle

```diff
@@ -1,10 +1,10 @@
 buildscript {
     repositories {
-        jcenter()
```

Comment:
I guess this should be merged to make PR https://github.com/owncloud/android/pull/2324 on main repo work